pyroclast/adversarial/adversarial.py

```diff
@@ -0,0 +1,48 @@
+import importlib
+import tensorflow as tf
+import matplotlib.pyplot as plt
+
+from pyroclast.common.tf_util import load_model
+from pyroclast.common.plot import plot_images
+from pyroclast.common.adversarial import fast_gradient_method
+from pyroclast.features.features import build_savable_objects
+
+
+def visualize_perturbation(data_dict, seed, output_dir, debug, module_name,
+                           model_name, norm, data_index, epsilon, **kwargs):
```

Comment:
Might doc this, especially the assumptions made about the module being imported.

Reply:
Also, the fact that this is attacking the usefulness of features

Reply:
What does `data_index` do?

Reply:
Going to take the index of data from the training data. If larger, test data. This is probably going to change. This file shouldn't be in question, I just needed a PR for the plot.